src/auth/auth.ts

```diff
@@ -99,7 +99,7 @@ export class AngularFireAuth {
             auth.useDeviceLanguage();
           }
           if (settings) {
-            auth.settings = settings;
+            auth.settings.appVerificationDisabledForTesting  = settings.appVerificationDisabledForTesting;
```

Comment:
Would be more flexible if we iterated over the entries, so we don't have to do anything if the Auth team adds more options in a future minor release.

Reply:
Done 🚀